.github/workflows/docker-push.yml

```diff
@@ -33,20 +19,19 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Build image
-        run: docker build . --file ./php-alpine/Dockerfile --build-arg COMPOSER_VERSION=$COMPOSER_VERSION --tag $IMAGE_NAME 
+        run: docker build . --file ./php-alpine/Dockerfile --build-arg COMPOSER_VERSION=$COMPOSER_VERSION --tag $IMAGE_NAME:latest
```

Comment:
Should we use tags like `latest`? We have a bot to take care of thing like this.

Reply:
Oh, it's a tag for our image, right? :v

Reply:
@krzysztofrewak No. Tag `latest` is temporary in workflow. At the end github actions is changing tag for image with command: `docker tag $IMAGE_NAME $IMAGE_ID:$VERSION`, where VERSION = tag of release.